tests/test_compute.py

```diff
@@ -23,14 +25,32 @@
     get_dataset_with_invalid_url_ddo,
     get_possible_compute_job_status_text,
     get_publisher_wallet,
-    get_nonce,
-    mint_tokens_and_wait,
-    send_order
-)
+    get_dataset_ddo_with_compute_service_no_rawalgo, get_dataset_ddo_with_compute_service_specific_algo_dids,
+    get_algorithm_ddo,
+    get_dataset_ddo_with_compute_service, get_compute_job_info, get_possible_compute_job_status_text,
+    mint_tokens_and_wait, get_nonce,
+    send_order)
 
 SERVICE_ENDPOINT = BaseURLs.BASE_PROVIDER_URL + '/services/download'
 
 
+def test_compute_expose_endpoints(client):
+    get_response = client.get('/')
+    result = get_response.get_json()
+    assert 'servicesEndpoints' in result
+    assert 'software' in result
+    assert 'version' in result
+    assert 'network-url' in result
+    assert 'provider-address' in result
+    assert get_response.status == '200 OK'
+    assert len(result['servicesEndpoints']) == len(
+        dict(map(lambda url: (url.endpoint.replace('services.', ''), '%s' % url),
```

Comment:
generating this dict can go into its own function, then can be called here in the endpoint above.